dags/openshift_nightlies/tasks/benchmarks/e2e.py

```diff
@@ -76,6 +76,14 @@ def __init__(self, dag, config: DagConfig, release: OpenshiftRelease, task_group
                 "AWS_DEFAULT_REGION": self.aws_creds['aws_region_for_openshift']
             }
     
+        if self.release.platform == "hypershift":
+            mgmt_cluster_name = release._generate_cluster_name()
+            self.env = {
+                "THANOS_RECEIVER_URL": var_loader.get_secret("thanos_receiver_url"),
+                "PROM_URL": var_loader.get_secret("thanos_querier_url"),
+                "MGMT_CLUSTER_NAME": f"{mgmt_cluster_name}.*",
```

Comment:
why the * ?

Reply:
Used in a wildcard query [here](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner/metrics-profiles/hypershift-metrics.yaml#L3)